docs/src/use/configure/combine-configs.md

```diff
@@ -116,4 +54,4 @@ export default defineConfig([
 ]);
 ```
 
-Here, each config object in `exampleConfigs` is applied only to files that match the pattern "`**/src/safe/*.js"`.
+Here, the `exampleConfigs` shareable configuration is applied to files that match the pattern "`**/*.js"` first and then another configuration object adds the desired configuration for `no-unused-vars`.
```

Comment:
Is the "another configuration object" the correct way to phrase this? I am unsure myself. Maybe adding the `rules` property is seen as merging the `exampleConfigs` object with its "outer" config object.

Reply:
I removed “another configuration object.” This should make it flow better.